samples/logging/README.rst

```diff
@@ -125,24 +125,40 @@ sample application (i.e., ``samples/logging``) and type:
 
 .. code-block:: console
 
-   $ west build -b nrf52840dk_nrf52840 samples/logging
+   $ west build -b nrf52840dk_nrf52840 samples/logging -p
    $ west flash
 
-nRF9160 Feather
----------------
+nRF9160 Feather or Thingy:91
+----------------------------
 
-On your host computer open a terminal window, locate the source code of this
-sample application (i.e., ``samples/logging``) and type:
+A full nRF9160 firmware `flashing overview`_ is available in the Golioth docs.
+Please use that resource to install the tools necessary to flash these devices.
+
+Configure the build for the MCUboot bootloader by adding this line to the
+configuration file (e.g. ``prj.conf``):
+
+.. code-block:: cfg
+
+   CONFIG_BOOTLOADER_MCUBOOT=y
+
+On your host computer open a terminal window, locate the directory that contains
+the sample folder (i.e., ``~/zephyr-nrf/modules/lib/golioth``) and type:
```

Comment:
It seems inconsistent to have that sentence only under `nRF9160 Feather or Thingy:91` subsection. In ALL cases we need to open a terminal change working directory.

> locate the directory that contains the sample folder (i.e., ``~/zephyr-nrf/modules/lib/golioth``)

This is the root directory of SDK repo. Sample folder is under `samples/<whatever>/`, so this is misleading now.

Reply:
Good point, this would make more sense higher in the file to apply to all hardware compiling examples.

I made this change because our build instructions pass the `samples/<whatever`. If user is already in the directory for that sample, build will fail:
```
ERROR: source directory samples/lightdb/get does not exist
FATAL ERROR: refusing to proceed without --force due to above error
```
I think it makes more sense to new users to build from the same root directory every time.

Reply:
> I think it makes more sense to new users to build from the same root directory every time.

Makes sense!